src/main/java/groovy/util/GroovyCollections.java

```diff
@@ -18,16 +18,19 @@
  */
 package groovy.util;
 
+import org.codehaus.groovy.runtime.ArrayUtil;
 import org.codehaus.groovy.runtime.DefaultGroovyMethods;
 import org.codehaus.groovy.runtime.ScriptBytecodeAdapter;
 import org.codehaus.groovy.runtime.typehandling.DefaultTypeTransformation;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
We prefer importing qualified classes and avoid using `*`

Reply:
I agree, i usually prefer it too but this it was my mistake because i had not adjusted the settings in my ide and it dit the * automatically. Feel free to CTRL + O to adjust the imports